banzai/stages.py

```diff
@@ -201,22 +201,28 @@ def apply_master_calibration(self, images, master_calibration_image, logging_tag
             bad_pixel_fraction /= noise
             bad_pixel_fraction = bad_pixel_fraction >= self.SIGNAL_TO_NOISE_THRESHOLD
             bad_pixel_fraction = bad_pixel_fraction.sum() / float(bad_pixel_fraction.size)
+            frame_is_bad = bad_pixel_fraction > self.ACCEPTABLE_PIXEL_FRACTION
 
             qc_results = {"master_comparison.fraction": bad_pixel_fraction,
                           "master_comparison.snr_threshold": self.SIGNAL_TO_NOISE_THRESHOLD,
-                          "master_comparison.pixel_threshold": self.ACCEPTABLE_PIXEL_FRACTION}
+                          "master_comparison.pixel_threshold": self.ACCEPTABLE_PIXEL_FRACTION,
+                          "master_comparison.failed": frame_is_bad,
+                          "master_comparison.last_good_master_filename": master_calibration_image.filename}
+
             for qc_check, qc_result in qc_results.items():
                 logs.add_tag(logging_tags, qc_check, qc_result)
+            logs.add_tag(logging_tags, 'filename', image.filename)
+            logs.add_tag(logging_tags, 'master_comparison_filename', master_calibration_image.filename)
+            self.logger.info("Performing comparison to last good master {caltype} frame".format(
+                caltype=self.calibration_type), extra=logging_tags)
```

Comment:
The indentation is funny here. Maybe make a message variable and them call the format on that in the logger.info?